### DISCUSSION TITLE
像油管那样网页全屏下鼠标往下滚直接滚到评论区

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#4977] [功能建议 / Ideas]
现在有不少脚本都能做到像油管那样网页全屏下鼠标往下滚直接滚到评论区
但是这些脚本包括油管自己本身也是，网页全屏并没有隐藏右侧的滚动条
导致观感非常不和谐

鉴于脚本组件里有个细滚动条组件，所以就在想又没办法做点文章隐藏掉滚动条的同时，保留直接滚动到评论区还有滚动调节音量的功能呢？

**Comment by Geeyun-JY3** [comment]
重复：#4180、#4697

**Comment by the1812** [comment]
隐藏滚动条可以用 chrome://flags/#overlay-scrollbars
